src/components/pages/BoatDetails.jsx

```diff
@@ -1,7 +1,92 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { getBoatDetails } from '../../redux/boatDetails/boatDetailsSlice';
 
-const BoatDetails = () => (
-  <div>Boat Details</div>
-);
+const BoatDetails = () => {
+  const boatDetailsState = useSelector((state) => state.boatDetails);
+  const dispatch = useDispatch();
+  const idBoat = useParams();
+
+  const boat = boatDetailsState.boatDetails;
+
+  const boatName = boat.name;
+
+  useEffect(() => {
+    dispatch(getBoatDetails(idBoat.id));
+  }, [dispatch]);
```

Comment:
There's an eslint warning for this: 
`WARNING in [eslint] 
src/components/pages/BoatDetails.jsx
  Line 17:6:  React Hook useEffect has a missing dependency: 'idBoat.id'. Either include it or remove the dependency array  react-hooks/exhaustive-deps`

Reply:
Thank you so much @hajnaloltyan !